upgrading/topics/keycloak/changes-19_0_0.adoc

```diff
@@ -111,6 +111,21 @@ Contains all REST endpoints that directly operate on the legacy store, and have
 These modules will be available as long as legacy stores will be supported.
 After that period, they will be removed.
 
+This change impacts deployments of existing user storage providers. If your user storage provider is deployed
```

Comment:
Adding the hint about the Wildfly distribution as users of the Quarkus distribution might not know what this is all about. I prefer "Wildfly distribution" over "legacy distribution", as "legacy" in this chapter usually refers to the "legacy store".

```suggestion
This change impacts deployments of existing user storage providers in the Wildfly distribution. If your user storage provider is deployed
```